Godeps/_workspace/src/github.com/prometheus/common/model/value.go

```diff
@@ -87,15 +82,15 @@ func (s *SamplePair) Equal(o *SamplePair) bool {
 	return s == o || (s.Value == o.Value && s.Timestamp.Equal(o.Timestamp))
 }
 
-func (s *SamplePair) String() string {
+func (s SamplePair) String() string {
 	return fmt.Sprintf("%s @[%s]", s.Value, s.Timestamp)
 }
 
 // Sample is a sample pair associated with a metric.
 type Sample struct {
-	Metric    Metric
-	Value     SampleValue
-	Timestamp Time
+	Metric    Metric      `json:"metric"`
```

Comment:
I see it was like that in the original version that got moved here. Still, seems like these are unused.